modules/client/src/internal/types.ts

```diff
@@ -237,3 +237,19 @@ export enum SubgraphPluginPermissionOperation {
   REVOKE = "Revoke",
   GRANT_WITH_CONDITION = "GrantWithCondition",
 }
+
+export enum ProposalActionTypes {
+  UPGRADE_TO = "upgradeTo",
+  UPGRADE_TO_AND_CALL = "upgradeToAndCall",
+  APPLY_UPDATE = "applyUpdate",
+  GRANT_PLUGIN_UPDATE_PERMISSION = "grantUpdatePluginPermission",
```

Comment:
```suggestion
  GRANT_PLUGIN_UPGRADE_PERMISSION = "grantUpgradePluginPermission",
```

Reply:
The permission is called `PLUGIN_UPDATE_PERMISSION`. I think it makes sense to have it like this

Reply:
We are calling it update everywhere, even the method in the contracts is `applyUpdate`

Reply:
But the permission name is UPGRADE_PLUGIN_PERMISSION (see https://github.com/aragon/osx/blob/c2be413a998c2995f4ed36ca648a5fd9b4284e77/packages/contracts/src/core/plugin/PluginUUPSUpgradeable.sol#L68)

Reply:
Things must match with the contracts - otherwise it gets confusing

Reply:
Okay Ill change it